draft-ietf-quic-multipath.md

```diff
@@ -1272,7 +1272,7 @@ with different delays explicitly.
 
 ## Packet Scheduling {#packet-scheduling}
 
-The transmission of data packets is limited
+The transmission of packets containing data frames is limited
 by the arrival of data from the application and by congestion control.
```

Comment:
Actually, that's true for not just data frames but also datagram frames, and in fact all frames not marked with the flag "C" in the table 3 of section 12.4 of RFC 9000. Only the ACK frames are not subject to congestion control. Not sure how to say that succintly. Maybe 

The transmission of ack-eliciting packets is limited